scripts/tests/scripts_regression_tests.py

```diff
@@ -315,9 +315,11 @@ def test_c_create_clone_keepexe(self):
             shutil.rmtree(testdir)
         prevtestdir = cls._testdirs[0]
         cls._testdirs.append(testdir)
+        user_mods_dir = os.path.join(CIME.utils.get_python_libs_root(), "..", "tests", "user_mods_test3")
 
-        run_cmd_assert_result(self, "%s/create_clone --clone %s --case %s --keepexe" %
```

Comment:
Did you mean to delete this old test? Now we don't have a test that shows that using the `--keep-exe` flag leads to a pass. Maybe that's okay: it's not clear to me that this old test was providing much benefit (e.g., it didn't seem to even check anything to confirm that `--keep-exe` worked properly). But I wanted to make sure that its replacement was intentional.

Reply:
@billsacks - @jedwards4b and I discussed this and decided that the next test captured the features of the old test as well as the new features.